zetaclient/orchestrator/orchestrator.go

```diff
@@ -661,7 +661,7 @@ func (oc *Orchestrator) ScheduleCctxSolana(
 // and optionally (de)provisions respective observers and signers.
 func (oc *Orchestrator) runObserverSignerSync(ctx context.Context) error {
 	// check every other zeta block
-	const cadence = 2 * evm.ZetaBlockTime
+	const cadence = 2 * constant.ZetaBlockTime
```

Comment:
Can we add a comment for the reasoning behind 2 .

I was also wondering if this should be increased to a higher number.

Reply:
Here is the improvement: https://github.com/zeta-chain/node/pull/2628/commits/3e42c21551c780f39862d17d3d38e5f8390763c8